pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/creator/VarByteChunkV4Test.java

```diff
@@ -46,73 +47,122 @@
 
 public class VarByteChunkV4Test {
 
-  private static final File TEST_DIR = new File(FileUtils.getTempDirectory(), "VarByteChunkV4Test");
+  private static File[] _dirs;
 
-  private File _file;
-
-  @DataProvider
+  @DataProvider(parallel = true)
```

Comment:
I'm not sure if `parallel = true` is discouraged

Reply:
I feel we can simplify the test by just checking whether the buffer file is created at desired location

Reply:
I think that testing in parallel is a better test, which would survive refactoring. Otherwise you're verifying the implementation in two places, and if you ever change the implementation, you lose the diagnostic power of the test.